client/dist/index.html

```diff
@@ -5,6 +5,7 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond&family=Nanum+Gothic+Coding&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Cute+Font&family=Nunito:wght@300&display=swap" rel="stylesheet">
```

Comment:
Adding more fonts is great and I approve. If you want to save a line of code, we can just add these new fonts to our existing link by just grabbing the parts we need from the query string. I'm pretty sure we can just take the `family=Cute+Font&family=Nunito:wght@300` and shove it inside the Google fonts link we already have on line 7.

No need to though-- this is 2-second job can wait ;)

Reply:
Adding more fonts is great and I approve. If you want to save a line of code, we can just add these new fonts to our existing link by just grabbing the parts we need from the query string. I'm pretty sure we can just take the family=Cute+Font&family=Nunito:wght@300 and shove it inside the Google fonts link we already have on line 7.

No need to though-- this is 2-second job can wait ;) <--- fixed